docroot/modules/custom/sitenow_dispatch/src/Form/ThankYouForm.php

```diff
@@ -176,6 +183,11 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       'includeBatchResponse' => FALSE,
     ];
 
+    // Appending a statement to the footer if approved to be included in promotional materials.
```

Comment:
```suggestion
    // Appending a statement to the footer if approved to be included in
    // promotional materials.
```
comment line 80 char limit